docker-compose.yml

```
@@ -14,6 +14,11 @@ services:
       - SECRET_KEY=${SECRET_KEY:-%9+dfjw+zd3@q)ehsx(pdwy-ueq0zt=90gb4+&qd^6ytpj@p7#}
       - DB_USER=${DB_USER:-openpersonen}
       - DB_PASSWORD=${DB_PASSWORD:-openpersonen}
+      - USE_STUF_BG_DATABASE=False
```

Minor, but in this case, `USE_STUF_BG_DATABASE` would always be `False` when using docker-compose
```suggestion
      - USE_STUF_BG_DATABASE=${USE_STUF_BG_DATABASE:-False}
```

---

Good point.  I initially thought the user would just change the docker-compose file but this is better.